decidim/decidim
# Remove invalid upload files

#### :tophat: What? Why?
This PR will remove any file uploaded, that is invalid ( either format or size ) 

#### Testing
1. Go to development_app's storage folder and remove all it's content (`./development_app/storage` )
2. Login as user, visit "my account"
3. On profile click to upload new photo 
4. Select an invalid file type (ex a pdf)
5. Check the storage folder and see the file is still there 
6. remove storage contents & apply patch 
7. Repeat 2,3,4
8. Check the storage folder and see there is no file left

:hearts: Thank you!
